recipes/autodiff/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set version = "0.4.0" %}
+
+package:
+  name: autodiff
+  version: {{ version }}
+
+source:
+  url: https://github.com/autodiff/autodiff/archive/v{{ version }}.tar.gz
+  sha256: 3f8246b8bc38bdfa41860c9a5e909b55793d2455edc525bdaa20b8060667b706
+
+build:
+  number: 0
+  skip: True  # [win and vc<15]
+  
+requirements:
+  build:
+    - {{ compiler('cxx') }}
```

`clang++` requires `clang` on macOS:
```suggestion
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
```

---

Great - thanks. Let's see now if it passes for osx.